### PR TITLE
Jee9beta

### PR DESCRIPTION
**Pull request by lauracowen**

### CODE REVIEW

File: posts/2020-06-22-jakarta-ee-9-open-liberty-beta.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+== Open Liberty betas
+
+When we first released the Open Liberty open source project, we focused mainly on adding production-quality capabilities to Liberty and releasing completed features. While we have often talked about our in-development work so that you know what's coming, the only way to get hold of this unfinished code to it try out was to take a look at our nightly builds. This is never ideal because nightly builds can be somewhat unstable, so we wanted to release a regular Open Liberty beta. 
+
+Open Liberty betas will contain everything in the most recent Open Liberty release plus features that are not yet finished. While these features are not yet suitable for use in production, we want to get feedback from our user community and them available to developers to use ahead of a production release. Of course, something appearing in a beta isn't a guarantee that it'll end up released: the feedback might be that it is a horrible, terrible, no good, very bad idea, and should be constrained to the dumpster of git history. At the very least, it might change a bit.
```
Comment (NottyCode) [inline posts/2020-06-22-jakarta-ee-9-open-liberty-beta.adoc:24]: I think the word make is missing from the following. Also to developers to doesn't sound write so I made a suggestion.

>While these features are not yet suitable for use in production, we want to get feedback from our user community and _make_ them available to developers ~to~ _for_ use ahead of a production release.

File: posts/2020-06-22-jakarta-ee-9-open-liberty-beta.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+== Betas and releases every four weeks
+
+We intend to release a new beta every four weeks at the same time as our normal four-weekly Open Liberty release. Usually, the beta will be available on the same day as the normal Open Liberty release, so the second beta, 20.0.0.8-beta, will be released on the same day as the 20.0.0.8 Open Liberty release. Today's beta (our first), however, is versioned as 20.0.0.7-beta but we've released it two weeks earlier than the Open Liberty 20.0.0.7 release.
```
Comment (NottyCode) [inline posts/2020-06-22-jakarta-ee-9-open-liberty-beta.adoc:28]: 20.0.0.8-beta is released at the same time as 20.0.0.7. The beta is one version ahead of the GA. Sorry this is on me. The 20.0.0.7-beta is available two weeks later than 20.0.0.6 which is when it should have come out.

Comment (lauracowen) [reply posts/2020-06-22-jakarta-ee-9-open-liberty-beta.adoc:28]: got it, thanks

File: posts/2020-06-22-jakarta-ee-9-open-liberty-beta.adoc (hunk 1 above)
Comment (lauracowen) [reply posts/2020-06-22-jakarta-ee-9-open-liberty-beta.adoc:24]: thx

**Pull request merged**